index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Dunk card display</title>
+		<link rel="stylesheet" type="text/css" href="quize.css">
+	</head>
+	<body>
+
+		<article1 class="box">
+			<section class="section1">
+				<h2>Yellow Dunk Cards</h2>
+			</section>
+
+			<section class="section2">
+
+				<img src="http://www.solidbackgrounds.com/images/125x125/125x125-banana-yellow-solid-color-background.jpg">
+
+				<p class="discription">
+					waistcoat single origin coffe
+					Waistcoat kitsch and single-origin coffee anddreamcatcher deep v,B hoodie echo park bushwick yr blog. Mixtape ethical tattooed dreamcatcher bicycle rights try-hard vegan, seitan plaid. Artisan mixtape austin street art.
+				</p>
+			</section>
+
```

Comment:
Requirement 4: "Each product should have three sections." That's 3 sections in addition to the 1 header mentioned before.
Requirement 5: "The first section should contain three child block elements."
5-i: The first element contains the product image. (So this should be a block level element that contains the image within)
5-ii: The second element contains the product description.
5-iii: The third element contains the product availability (e.g. "Available" or "Not Available")